eval/eval.go

```diff
@@ -150,6 +150,90 @@ func New(path string) *Program {
 	})
 	addUniverse("make", p.builtinMake)
 	addUniverse("new", p.builtinNew)
+	addUniverse("complex", func(re, im interface{}) interface{} {
```

Comment:
As there's a bit of code here, how about defining it in a method, `builtinComplex`?

Reply:
done.